src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectPropertyDescriptor.cs

```diff
@@ -396,6 +396,8 @@ private MethodInfo ResetMethodValue
         /// </summary>
         private MethodInfo SetMethodValue
         {
+            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2075:UnrecognizedReflectionPattern",
+                Justification = "_componentClass is annotated with All, but the trimmer is still warning. https://github.com/mono/linker/issues/1882")]
```

Comment:
Is'nt the warning because what you are calling GetProperty on is not _componentClass but it's BaseType instead?

Reply:
Yes - https://github.com/mono/linker/issues/1882 is about propagating the `All` annotation to over the `BaseType` as well (Since trimmer will preserve everything on base types as well).

Reply:
I updated the Justification to better reflect this.